qCC/ccColorFromScalarDlg.cpp

```diff
@@ -420,6 +420,10 @@ void ccColorFromScalarDlg::onApply()
 					col[i] = 255 - col[i];
 				}
 			}
+			if (!m_cloud->hasColors())
```

Comment:
Why not calling it at the beginning of the "onApply" method? Since the other case below ('else' case) has the same issue.

Reply:
I totally missed that, thanks!
https://github.com/CloudCompare/CloudCompare/pull/1114